apps/token/src/routes/tranches/tranche-progress.tsx

```diff
@@ -28,7 +29,7 @@ export const TrancheProgress = ({
         <span className="tranches__progress-title">{t('Locked')}</span>
         <ProgressBar
           width={220}
-          color={Colors.PINK}
+          color={Colors.pink}
```

Comment:
suggestion (non-blocking): Can we use a tailwind class here?

Reply:
I can use the tailwind class only if I change the progress bar component to use tailwind. Should I update ProgressBar component as part of this PR?

Reply:
This would require changing ProgressBar to use tailwind, and I am not sure if I should do this change as part of this PR. I know @sam-keen is working on updating the [token app to use tailwind](https://github.com/vegaprotocol/frontend-monorepo/issues/304) atm.